src/java.base/share/classes/jdk/internal/classfile/components/ClassRemapper.java

```diff
@@ -0,0 +1,401 @@
+import jdk.internal.classfile.ClassModel;
+import jdk.internal.classfile.ClassSignature;
+import jdk.internal.classfile.ClassTransform;
+import jdk.internal.classfile.Classfile;
+import jdk.internal.classfile.CodeBuilder;
+import jdk.internal.classfile.CodeElement;
+import jdk.internal.classfile.CodeModel;
+import jdk.internal.classfile.CodeTransform;
+import jdk.internal.classfile.FieldBuilder;
+import jdk.internal.classfile.FieldElement;
+import jdk.internal.classfile.FieldModel;
+import jdk.internal.classfile.FieldTransform;
+import jdk.internal.classfile.Interfaces;
+import jdk.internal.classfile.MethodBuilder;
+import jdk.internal.classfile.MethodElement;
+import jdk.internal.classfile.impl.TemporaryConstantPool;
+import jdk.internal.classfile.instruction.FieldInstruction;
+import jdk.internal.classfile.instruction.InvokeDynamicInstruction;
+import jdk.internal.classfile.instruction.InvokeInstruction;
+import jdk.internal.classfile.instruction.NewMultiArrayInstruction;
+import jdk.internal.classfile.instruction.NewObjectInstruction;
+import jdk.internal.classfile.instruction.NewReferenceArrayInstruction;
+import jdk.internal.classfile.instruction.TypeCheckInstruction;
+import jdk.internal.classfile.MethodModel;
+import jdk.internal.classfile.MethodSignature;
+import jdk.internal.classfile.MethodTransform;
+import jdk.internal.classfile.Signature;
+import jdk.internal.classfile.Superclass;
+import jdk.internal.classfile.TypeAnnotation;
+import jdk.internal.classfile.attribute.EnclosingMethodAttribute;
+import jdk.internal.classfile.attribute.ExceptionsAttribute;
+import jdk.internal.classfile.attribute.InnerClassInfo;
+import jdk.internal.classfile.attribute.InnerClassesAttribute;
+import jdk.internal.classfile.attribute.ModuleAttribute;
+import jdk.internal.classfile.attribute.ModuleProvideInfo;
+import jdk.internal.classfile.attribute.RecordAttribute;
+import jdk.internal.classfile.attribute.RecordComponentInfo;
+import jdk.internal.classfile.attribute.RuntimeInvisibleAnnotationsAttribute;
+import jdk.internal.classfile.attribute.RuntimeInvisibleParameterAnnotationsAttribute;
+import jdk.internal.classfile.attribute.RuntimeInvisibleTypeAnnotationsAttribute;
+import jdk.internal.classfile.attribute.RuntimeVisibleAnnotationsAttribute;
+import jdk.internal.classfile.attribute.RuntimeVisibleParameterAnnotationsAttribute;
+import jdk.internal.classfile.attribute.RuntimeVisibleTypeAnnotationsAttribute;
+import jdk.internal.classfile.attribute.SignatureAttribute;
+import jdk.internal.classfile.constantpool.Utf8Entry;
+import jdk.internal.classfile.instruction.ExceptionCatch;
+import jdk.internal.classfile.instruction.LocalVariable;
+import jdk.internal.classfile.instruction.LocalVariableType;
+import jdk.internal.classfile.impl.Util;
+import jdk.internal.classfile.instruction.ConstantInstruction.LoadConstantInstruction;
+
+/**
+ * ClassRemapper is a {@link jdk.internal.classfile.ClassTransform}, {@link jdk.internal.classfile.FieldTransform},
```

Comment:
Maybe wrap occurrences of `ClassRemapper` with `{@code}` (here and elsewhere)

Reply:
will fix it, thanks.